eventdata/runners/createindex_runner.py

```diff
@@ -22,8 +25,18 @@ def createindex(es, params):
         else:
         	template_name = 'elasticlogs'
         
-        if logger.isEnabledFor(logging.DEBUG):
-            logger.debug("[createindex] Upload index template {} => {}".format(template_name, json.dumps(params['index_template_body'])))
+        if 'mappings' in params['index_template_body'] and isinstance(params['index_template_body']['mappings'], str):
+            if params['index_template_body']['mappings'] in gs.global_config.keys():
+                mapping = gs.global_config[params['index_template_body']['mappings']]
+            else:
+                mapping_path = os.path.expandvars(params['index_template_body']['mappings'])
+                logger.info("[createindex] Use mapping file: {}".format(mapping_path))
```

Comment:
Nit: For logging calls it is more idiomatic to use `%s` and provide the format parameters as a comma-separated list (i.e. `logger.info("[createindex] Use mapping file: %s", mapping_path)`)

The reason is that if the statement would not be logged anyway there is no need to format the logging string.